### PR TITLE
add aocbot

### PR DESCRIPTION
**Pull request by julian9499**

### CODE REVIEW

File: apps/aocbot/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      maxUnavailable: 0
+  template:
+    metadata:
+      name: aocbot
+      labels:
+        app: aocbot
+    spec:
+      containers:
+        - name: aocbot
+          image: ghcr.io/wisvch/aocbot:20241205-287e5bf # {"$imagepolicy": "flux-system:aoch"}
```
Comment (dsluijk) [inline apps/aocbot/deploy.yaml:26]: Perhaps change the image policy name, unless you want the other leaderboard

**Review by dsluijk — CHANGES_REQUESTED** [review]
You also need to add a reference to this folder in the main customization file in the apps folder

File: apps/aocbot/deploy.yaml (hunk 1 above)
Comment (julian9499) [reply apps/aocbot/deploy.yaml:26]: Good one on it

Comment (julian9499) [reply apps/aocbot/deploy.yaml]: Done

**Pull request merged**